src/Fantomas.Core/SourceParser.fs

```diff
@@ -1595,7 +1595,9 @@ let (|AnonRecordFieldName|) (ident: Ident, eq: range option, e: SynExpr) =
     let range = Range.unionRanges ident.idRange e.Range
     (ident, eq, e, range)
 
-let (|AnonRecordFieldType|) (ident, t: SynType) = (ident, t)
+let (|AnonRecordFieldType|) (ident: Ident, ty: SynType) =
+    let range = Range.unionRanges ident.idRange ty.Range
```

Comment:
Any reason you are adding this range? It seems to be ignored in `genAnonRecordFieldType`.

Reply:
I was trying to fix an issue I ran into and thought it might be fixable by using that somehow, but it turned out not to be the case. Reverting this.